lib/index.js

```diff
@@ -37,7 +38,8 @@ class ElectronDownloader {
   }
 
   get cache () {
-    return this.opts.cache || path.join(homePath(), './.electron')
+    // use passed argument or XDG environment variable fallback to OS default
+    return this.opts.cache || envPaths('electron-download', {suffix: ''}).cache
```

Comment:
Yeah, ultimately the `electron` module uses this. Now that you've reminded me, it's worth documenting where the default cache directory is on Windows/macOS/Linux in the README.